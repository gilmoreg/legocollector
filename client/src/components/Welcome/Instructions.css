.Instructions {
    position: fixed;
    bottom: 110px;
    right: 25px;
    border-radius: 5px;
    background-color: rgba(255,246,229,0.7);
    padding: 1rem;
}

.Instructions .plus {
    display: inline-block;
    border: 0.5rem solid #FF7F66;
    border-radius: 100%;
    background-color: #FF7F66;
    color: white;
    width: 2rem;
    height: 2rem;
    font-size: 14px;
}