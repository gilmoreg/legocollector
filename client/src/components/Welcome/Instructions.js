import React from 'react';
import styled from 'styled-components';

const StyledInstructions = styled.div`
  position: fixed;
  bottom: 110px;
  right: 25px;
  border-radius: 5px;
  background-color: rgba(255,246,229,0.7);
  padding: 1rem;
  z-index: -1;
`;

const StyledPlus = styled.div`
  display: inline-block;
  border: 0.5rem solid #FF7F66;
  border-radius: 100%;
  background-color: #FF7F66;
  color: white;
  width: 2rem;
  height: 2rem;
  font-size: 14px;
`;

const Instructions = () => (
  <StyledInstructions>
    Click on the <StyledPlus>+</StyledPlus> to start watching sets!
  </StyledInstructions>
);

export default Instructions;
