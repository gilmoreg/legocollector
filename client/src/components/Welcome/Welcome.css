.Welcome {
  background-color: rgba(255,246,229,0.7);
  border-radius: 5px;
  margin: 1rem;
  padding: 1rem;
}

.Welcome p {
  text-align: left;
}