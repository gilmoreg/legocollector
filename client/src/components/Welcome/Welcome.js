import React from 'react';
import './Welcome.css';

const Welcome = () => (
  <div className="Welcome">
    Hi
  </div>
);

export default Welcome;
