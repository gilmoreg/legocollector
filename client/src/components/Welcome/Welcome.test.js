import React from 'react';
import { shallow } from 'enzyme';
import Welcome from './Welcome';

it('renders without crashing', () => {
  const wrapper = shallow(
    <Welcome />,
  );
  expect(wrapper).toMatchSnapshot();
});
