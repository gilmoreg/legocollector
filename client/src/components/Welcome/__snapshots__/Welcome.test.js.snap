// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Welcome />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Welcome to Lego Collector!
        </h2>,
        <p>
          If you collect Lego sets for profit, you always want to know when a set is about to run out of stock. Lego Collector keeps you up to date by tracking Amazon's stock levels for Lego sets you specify.
        </p>,
        <p>
          Check the stock levels of all the sets you have your eye on in one convenient place!
        </p>,
        <p>
          Soon it will be able to notify you via email when a certain stock falls below a threshold you define so you can snap up one of the last sets before it's too late!
        </p>,
      ],
      "className": "Welcome",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Welcome to Lego Collector!",
        },
        "ref": null,
        "rendered": "Welcome to Lego Collector!",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "If you collect Lego sets for profit, you always want to know when a set is about to run out of stock. Lego Collector keeps you up to date by tracking Amazon's stock levels for Lego sets you specify.",
        },
        "ref": null,
        "rendered": "If you collect Lego sets for profit, you always want to know when a set is about to run out of stock. Lego Collector keeps you up to date by tracking Amazon's stock levels for Lego sets you specify.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Check the stock levels of all the sets you have your eye on in one convenient place!",
        },
        "ref": null,
        "rendered": "Check the stock levels of all the sets you have your eye on in one convenient place!",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Soon it will be able to notify you via email when a certain stock falls below a threshold you define so you can snap up one of the last sets before it's too late!",
        },
        "ref": null,
        "rendered": "Soon it will be able to notify you via email when a certain stock falls below a threshold you define so you can snap up one of the last sets before it's too late!",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Welcome to Lego Collector!
          </h2>,
          <p>
            If you collect Lego sets for profit, you always want to know when a set is about to run out of stock. Lego Collector keeps you up to date by tracking Amazon's stock levels for Lego sets you specify.
          </p>,
          <p>
            Check the stock levels of all the sets you have your eye on in one convenient place!
          </p>,
          <p>
            Soon it will be able to notify you via email when a certain stock falls below a threshold you define so you can snap up one of the last sets before it's too late!
          </p>,
        ],
        "className": "Welcome",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to Lego Collector!",
          },
          "ref": null,
          "rendered": "Welcome to Lego Collector!",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "If you collect Lego sets for profit, you always want to know when a set is about to run out of stock. Lego Collector keeps you up to date by tracking Amazon's stock levels for Lego sets you specify.",
          },
          "ref": null,
          "rendered": "If you collect Lego sets for profit, you always want to know when a set is about to run out of stock. Lego Collector keeps you up to date by tracking Amazon's stock levels for Lego sets you specify.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Check the stock levels of all the sets you have your eye on in one convenient place!",
          },
          "ref": null,
          "rendered": "Check the stock levels of all the sets you have your eye on in one convenient place!",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Soon it will be able to notify you via email when a certain stock falls below a threshold you define so you can snap up one of the last sets before it's too late!",
          },
          "ref": null,
          "rendered": "Soon it will be able to notify you via email when a certain stock falls below a threshold you define so you can snap up one of the last sets before it's too late!",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
